php/bootstrap.php

```diff
@@ -15,24 +15,25 @@
  */
 function get_bootstrap_steps() {
 	return [
-		'WP_CLI\Bootstrap\DeclareFallbackFunctions',
-		'WP_CLI\Bootstrap\LoadUtilityFunctions',
-		'WP_CLI\Bootstrap\LoadDispatcher',
-		'WP_CLI\Bootstrap\DeclareMainClass',
-		'WP_CLI\Bootstrap\DeclareAbstractBaseCommand',
-		'WP_CLI\Bootstrap\IncludeFrameworkAutoloader',
-		'WP_CLI\Bootstrap\ConfigureRunner',
-		'WP_CLI\Bootstrap\InitializeColorization',
-		'WP_CLI\Bootstrap\InitializeLogger',
-		'WP_CLI\Bootstrap\DefineProtectedCommands',
-		'WP_CLI\Bootstrap\LoadExecCommand',
-		'WP_CLI\Bootstrap\LoadRequiredCommand',
-		'WP_CLI\Bootstrap\IncludePackageAutoloader',
-		'WP_CLI\Bootstrap\IncludeFallbackAutoloader',
-		'WP_CLI\Bootstrap\RegisterFrameworkCommands',
-		'WP_CLI\Bootstrap\RegisterDeferredCommands',
-		'WP_CLI\Bootstrap\InitializeContexts',
-		'WP_CLI\Bootstrap\LaunchRunner',
+		Bootstrap\DeclareFallbackFunctions::class,
+		Bootstrap\LoadUtilityFunctions::class,
+		Bootstrap\LoadDispatcher::class,
+		Bootstrap\DeclareMainClass::class,
+		Bootstrap\DeclareAbstractBaseCommand::class,
+		Bootstrap\IncludeFrameworkAutoloader::class,
+		Bootstrap\ConfigureRunner::class,
+		Bootstrap\ExtractDefaultCaCertificate::class,
```

Comment:
It is a bit obfuscated as I switched all of the class names to the safer `::class` notation, but this is the one that was added:
```
Bootstrap\ExtractDefaultCaCertificate::class
```